package/autoyast2.spec

```diff
@@ -303,6 +298,9 @@ rmdir $RPM_BUILD_ROOT/%{_prefix}/share/doc/packages/autoyast2/html/autoyast
 %dir %{yast_libdir}/autoinstall/dialogs
 %{yast_libdir}/autoinstall/dialogs/*.rb
 
+%{yast_libdir}/autoinstall/widgets
+%{yast_libdir}/autoinstall/presenters
+
```

Comment:
This is done in a different way than the `dialogs` and `clients` directories, that have a `dir` entry for the dir and then the `*.rb` line for the files. I don't know if any of the methods is better than the other, but the difference looks strange.

Reply:
I realize about that, but instead of "duplicating" the lines that already are in the src/Makefile.am, I preferred to wait for comments in the PR review. And it worked.

Reply:
As discussed in IRC, let's keep it as it is.

Using the @jreidinger words, 

>  as general rule use minimal working one. Only advantage of two lines approach is that it catches for you if you add to git some thrash like y2log file as it does not match *.rb pattern